database/insertData/03_template_E_coreActions.js

```diff
@@ -0,0 +1,211 @@
+/*
+TEMPLATE E - Core Actions
+  - empty template that lists all of the core actions
```

Comment:
Is this just intended as a reference for now?
We had the idea to save all the core actions in a GraphQL fragment (#184) and then every template could just slug this Fragment into it's mutation. It's probably a good idea to do that now, right, or at least soon after this issue -- then we can remove the need for this file.

Reply:
ok good point, i'll do that now

Reply:
actually, this would require a bit more thought.

* dont' think you can use fragments for mutations
* if you could we would need to concat the fragment definition to the end of each query (apollo client does this automatically)
* would be a bit harder if we added custom actions to know next sequence number (but I guess we can start them at like 101)

Reply:
discussion to continue on that issue if it's ok